text/0750-feature-flag-advisor.md

```diff
@@ -0,0 +1,183 @@
+
+This problem is more severe for users who create their CDK projects without using `cdk init` because their feature flags have not been configured at all. This results in different app behavior and missing out on potential bug fixes. By providing a straightforward `cdk flags` command to find, modify, and view their feature flag configuration, we reduce the burden on the user to maintain upkeep of their applications.
+
+## Why should we do this?
+
+Feature flags have the potential to improve the user experience by incorporating bug fixes and new default behavior, so it is beneficial to create a visible tool for users to increase autonomy within their CDK projects. This tool will also reduce burden on the user by allowing them to change their feature flag configurations through simple commands on their CLI. By creating a Feature Flag CLI tool, we show users with information on their feature flag configuration, inform them about available feature flags, and allow them to modify their initial feature flag states seamlessly, resulting in a better user experience.
+
+## Why should we *not* do this?
+
+This tool aims to simplify the feature flag configuration file by asking users if they would like to make the changes from the CLI. However, this creates a risk of users applying our recommendations without fully understanding its impact on their application. For example, a user could unknowingly enable a recommended flag that impacts behavior a user is depending on. To ensure they are informed about these changes as much as possible, any change will result in a comparison of their CloudFormation template and their modified one.
+
+Adding a new CLI tool also increases the CDK CLI complexity, requiring more maintenance and support for this new feature. 
+
+## What alternative solutions did you consider?
+
+Instead of creating a Feature Flag CLI tool, we could expand on `cdk synth`. If a user’s feature flags differ from our recommended values, it would display a warning or suggestion. This reduces the need for another CLI command, but this method risks negatively impacting user experience due to cluttering the `cdk synth` output if many flags differ. 
```

Comment:
Having `cdk synth` do something flag-related is 100% in the design here, and is not Future Work (and also not really an "alternative")